front/components/assistant/AssistantDetails.tsx

```diff
@@ -274,13 +276,15 @@ function ButtonsSection({
 
       {canDelete && (
         <>
-          <DeletionModal
+          <DeleteAssistantDialog
             owner={owner}
-            agentConfiguration={agentConfiguration}
+            agentConfigurationId={agentConfiguration.sId}
```

Comment:
nit: why not passing the typed object?

Reply:
Because the AssistantBuilder does not have the full AgentconfigurationType